api/http/proxy/containers.go

```diff
@@ -62,27 +62,27 @@ func containerInspectOperation(response *http.Response, executor *operationExecu
 
 	containerID := responseObject[containerIdentifier].(string)
 	responseObject, access := applyResourceAccessControl(responseObject, containerID, executor.operationContext)
-	if !access {
-		return rewriteAccessDeniedResponse(response)
+	if access {
```

Comment:
Question: why do you changed the access check here to check that access is authorized instead of access denied? I see that you changed this for all the resources.

Reply:
I changed `applyResourceAccessControlFromLabel` and `applyResourceAccessControl` and now access is denied by default.

Reply:
OK.